resources/js/Components/Collections/CollectionsGrid/CollectionsGrid.test.tsx

```diff
@@ -90,19 +90,19 @@ describe("CollectionsGrid", () => {
         const values = getAllByTestId("CollectionFloorPrice");
 
         const sorted: string[] = [
-            "0.7 ETH",
+            "0.1 ETH",
+            "0.2 ETH",
             "0.4 ETH",
-            "700 USDC",
+            "0.7 ETH",
             "0.3 ETH",
+            "0.1 ETH",
+            "0.05 ETH",
+            "100 USDC",
+            "200 USDC",
             "400 USDC",
-            "0.2 ETH",
+            "700 USDC",
             "300 USDC",
-            "200 USDC",
-            "1 ETH",
-            "1 ETH",
-            "100 USDC",
             "100 USDC",
-            "0.05 ETH",
             "50 USDC",
```

Comment:
I'm confused about this "sorted" order as it doesn't seem sorted by value 🤔 care to explain what is expected here in terms of sorting? To me it seems to be identical to the original array values and not sorted:

```
const ethereumValues = [0.1, 0.2, 0.4, 0.7, 0.3, 0.1, 0.05, null, null];
const usdcValues = [100, 200, 400, 700, 300, 100, 50, null, null];
```

Reply:
but if it's sorted by value and value is floor price * currency * nft amount, where nfts are always 1, shouldn't the same eth values be together? now it's:
```
"0.1 ETH", // -> 0.1
"0.2 ETH",
"0.4 ETH",
"0.7 ETH",
"0.3 ETH",
"0.1 ETH", // -> also 0.1 ?
 "0.05 ETH",
 ```
 
 which makes it hard to read what this test is really doing. Can you add a comment with the values behind it so it's easier to see the sorting order that it tests against?